compiler/compile/OMRCompilation.hpp

```diff
@@ -621,7 +621,7 @@ class OMR_EXTENSIBLE Compilation
    uint16_t getMaxInlineDepth() {return _maxInlineDepth;}
    bool incInlineDepth(TR::ResolvedMethodSymbol *, TR::Node *callNode, bool directCall, TR_VirtualGuardSelection *guard, TR_OpaqueClassBlock *receiverClass, TR_PrexArgInfo *argInfo = 0);
    bool incInlineDepth(TR::ResolvedMethodSymbol *, TR_ByteCodeInfo &, int32_t cpIndex, TR::SymbolReference *callSymRef, bool directCall, TR_PrexArgInfo *argInfo = 0);
-   bool incInlineDepth(TR_OpaqueMethodBlock *, TR::ResolvedMethodSymbol *, TR_ByteCodeInfo &, TR::SymbolReference *callSymRef, bool directCall, TR_PrexArgInfo *argInfo = 0);
+   bool incInlineDepth(TR_OpaqueMethodBlock *, TR::ResolvedMethodSymbol *, TR_ByteCodeInfo &, TR::SymbolReference *callSymRef, bool directCall, TR_PrexArgInfo *argInfo = 0, TR_AOTMethodInfo *aotMethodInfo = 0);
```

Comment:
There doesn't seem to be any place where we call this API without `argInfo` so I don't think we should have any default parameters for this API; in fact it might even worth be making this particular API protected (since it's only called by the other `incInlineDepth` methods.

Reply:
I made it protected, however since there are already multiple protected blocks in the compilation class I just made another one, just for `incInlineDepth` to avoid moving it hundreds of lines below. I'll change that if you don't think that's good